features/events/syslog/src/main/resources/META-INF/opennms/applicationContext-syslogDaemon.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:tx="http://www.springframework.org/schema/tx"
+  xmlns:context="http://www.springframework.org/schema/context"
+  xmlns:util="http://www.springframework.org/schema/util"
+  xmlns:onmsgi="http://xmlns.opennms.org/xsd/spring/onms-osgi"
+  xsi:schemaLocation="
+  http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.0.xsd
+  http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd
+  http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx-4.0.xsd
+  http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util-4.0.xsd
+  http://xmlns.opennms.org/xsd/spring/onms-osgi http://xmlns.opennms.org/xsd/spring/onms-osgi.xsd
+">
+
+  <context:annotation-config />
+  <tx:annotation-driven/>
+
+  <bean id="executorFactoryJava" class="org.opennms.core.concurrent.ExecutorFactoryJavaImpl"/>
+
+  <bean id="executorFactoryCassandraSEP" class="org.opennms.core.concurrent.cassandra.ExecutorFactoryCassandraSEPImpl"/>
+
+  <bean id="syslogReceiverNio" class="org.opennms.netmgt.syslogd.SyslogReceiverNioThreadPoolImpl">
+    <constructor-arg ref="syslogdConfigFactory"/>
+  </bean>
+
+  <bean id="syslogReceiverNioDisruptor" class="org.opennms.netmgt.syslogd.SyslogReceiverNioDisruptorImpl">
+    <constructor-arg ref="syslogdConfigFactory"/>
+  </bean>
+
+  <bean id="syslogReceiverNioDisruptorSEP" class="org.opennms.netmgt.syslogd.SyslogReceiverNioDisruptorImpl">
+    <constructor-arg ref="syslogdConfigFactory"/>
+    <property name="executorFactory" ref="executorFactoryCassandraSEP"/>
+  </bean>
+
+  <bean id="syslogReceiverJavaNet" class="org.opennms.netmgt.syslogd.SyslogReceiverJavaNetImpl">
+    <constructor-arg ref="syslogdConfigFactory"/>
+  </bean>
+
+  <bean id="syslogReceiverCamelNetty" class="org.opennms.netmgt.syslogd.SyslogReceiverCamelNettyImpl">
+    <constructor-arg ref="syslogdConfigFactory"/>
+  </bean>
+
+  <!-- Expose one particular implementation as the chosen implementation -->
```

Comment:
This was just used for some testing so far. I'll be refactoring this as we modularize things further for Minion. Work in progress! :)